src/resources/postcss/tickets-commerce/admin/orders/single.pcss

```
@@ -1,3 +1,59 @@
+.tec-tickets-commerce-vertical-form label {
+	display: block;
+	margin-bottom: 2px;
+	margin-top: 2px;
+}
+.tec-tickets-commerce-vertical-form .tec-tickets-commerce-row {
+	display: block;
+	padding: 2px 0;
+}
+#tribe-tickets-order-details .tec-tickets-commerce-error-message {
+	color: var(--tec-error-border-color);
+}
+.tec-tickets-commerce-vertical-form .tec-tickets-commerce-error-field {
+	border-color: var(--tec-error-border-color);
+}
+#tribe-tickets-order-details .tribe-dialog__wrapper .tribe-common-c-loader {
+	background: rgba(255, 255, 255, 0.8);
+	bottom: 0;
+	height: 100%;
+	justify-content: center;
+	left: 0;
+	padding: 0;
+	position: absolute;
+	right: 0;
+	top: 0;
+	width: 100%;
+	z-index: 10;
+}
+
+#tribe-tickets-order-details .tribe-dialog__wrapper.tribe-modal__wrapper {
+	border-radius: 0;
+	padding: 0;
+}
+
+#tribe-tickets-order-details .tribe-dialog__wrapper.tribe-modal__wrapper h2.tribe-dialog__title {
+	border-bottom: 1px solid;
+	font-size: var(--tec-font-size-3);
+	margin: 0;
+	padding: 2px 16px;
+	width: 100%;
+}
+
+#tribe-tickets-order-details .tribe-dialog__wrapper .tribe-dialog__close-button.tribe-modal__close-button {
+	float: right;
+	margin-top: 6px;
+	margin-right: 16px;
+}
+
+#tribe-tickets-order-details .tribe-dialog__content.tribe-modal__content {
+	padding: 16px;
```

Use variables where possible:
```suggestion
	padding: var(--tec-spacer-3);
```